src/app/admin/messages/(components)/msg-skeleton.tsx

```
@@ -0,0 +1,19 @@
+import { nanoid } from 'nanoid';
+import React from 'react';
+
+export default function MessageSkeleton() {
+  return (
+    <div className="space-y-4">
+      {[...Array(4)].map(() => (
+        <div key={nanoid()} className="animate-pulse rounded-lg border p-4">
+          <div className="mb-2 h-6 rounded bg-gray-300"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)